### PR TITLE
Changed the structure, added tox and travis configuration.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
@gainward, can you take a look?

File: README.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 
-To run the test, first please set the GOOGLE_APPENGINE_SDK_PATH.
+To run the tests, please install App Engine Python SDK and tox and run
+tox with the environment variable PYTHONPATH to the App Engine Python SDK.
 
-  $ export GOOGLE_APPENGINE_SDK_PATH=/somewhere/google_appengine
+You can install App Engine Python SDK with Google Cloud SDK[1] with the following command:
 
-Then run `scripts/run-test.py` as follows.
+  $ gcloud components update gae-python
```
Comment (gainward) [inline README.md]: think this needs to be indented 4 spaces to be marked as code

File: README.md [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 
-  $ scripts/run-test.py test
+Here is instructions to run the tests with virtualenv, $GCLOUD is your
+Google Cloud SDK installation path.
+
+  $ virtualenv -p python2.7 --no-site-packages .
```
Comment (gainward) [inline README.md]: same as above

File: app.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+application: ndb-snippets
+version: 1
+api_version: 1
+runtime: python27
+threadsafe: true
+
+handlers:
+- url: /.*
+  script: main.application
```
Comment (gainward) [inline app.yaml]: main.app?

File: README.md (hunk 1 above)
Comment (tmatsuo) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (tmatsuo) [inline README.md]: Done

File: app.yaml (hunk 3 above)
Comment (tmatsuo) [inline app.yaml]: Replaced it with a static handler which returns README.md.

**Comment by tmatsuo** [comment]
@gainward thanks! PTAL

**Comment by gainward** [comment]
LGTM

**Pull request merged**